src/plots/cartesian/dragbox.js

```diff
@@ -296,7 +296,7 @@ module.exports = function dragBox(gd, plotinfo, x, y, w, h, ns, ew) {
         dragTail(zoomMode);
 
         if(SHOWZOOMOUTTIP && gd.data && gd._context.showTips) {
-            Lib.notifier('Double-click to<br>zoom back out', 'long');
+            Lib.notifier(Lib._(gd, 'Double-click to<br>zoom back out'), 'long');
```

Comment:
Seems funny to include `<br>` in the translation - should we 🔪  that and maybe just set notifier max width in css? (it's a `<div>` so we can do that 😄 )

Reply:
> should we 🔪 that and maybe just set notifier max width in css?

Actually we already have it, and as currently set we just break at the next word:
![screen shot 2017-12-07 at 2 51 29 pm 2](https://user-images.githubusercontent.com/2678795/33735788-57187ade-db5e-11e7-9812-8273b970978b.png)

I'm inclined to take the `<br>` out here and just leave it to break where it will. We can of course still use `<br>` in cases where it conveys some meaning, but this is not such a case.

Reply:
`<br>` removed in https://github.com/plotly/plotly.js/pull/2195/commits/5d6f26772191935a6026f0acad7efaa2b90d2dea